<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCornerDetection</class>
 <widget class="QDialog" name="DialogCornerDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QRadioButton" name="rBShiTomasi">
     <property name="text">
      <string>Shi Tomasi</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>blocksize</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="rBHarris">
     <property name="text">
      <string>Harris</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>sobelsize</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="sBBlockSize">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="sBSobelSize">
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="13">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="10" column="2">
    <widget class="QSlider" name="sliderColor">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>color</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>line thickness</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>quality</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="sliderQuality">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="cBxBorderType">
     <item>
      <property name="text">
       <string>constant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>replicate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>reflect_101</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" rowspan="5" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>bordertype</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QCheckBox" name="cBDrawCorners">
     <property name="text">
      <string>draw Corners</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QSlider" name="sliderLineThickness">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
